### PR TITLE
Feature/dx 502 mobile pay after payment

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: payments/mobile-pay/after-payment.md [hunk 1]
```diff
@@ -16,11 +16,216 @@ sidebar:
 
 {% include alert-development-section.md %}
 
-### Options after posting a payment
+## Payment Resource
+
+### Create Payment
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]: Let's have the "Create Payment" operation in "Other Features" for now.

File: payments/mobile-pay/after-payment.md [hunk 2]
```diff
@@ -32,66 +32,20 @@ sidebar:
...
+operation.
 
 {:.code-header}
 **Request**
 
 ```http
-POST /psp/mobilepay/payments/e7919b4f-81a2-4ffb-ec40-08d617d580a2/captures HTTP/1.1
-Host: {{ page.apiUrl }}
+POST /psp/mobilepay/payments/{{ page.paymentId }}/captures HTTP/1.1
+Host: api.externalintegration.payex.com
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]:
```suggestion
Host: {{ page.apiUrl }}
```

File: _includes/transaction-response.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "payment": "/psp/{{ instrument }}/payments/{{ page.paymentId }}",
+    "{{ transaction | append: "s" }}": {
```
Comment (asbjornu) [inline _includes/transaction-response.md]: Perhaps make a `plural` variable here as you did in #114?

File: payments/mobile-pay/after-payment.md [hunk 4]
```diff
@@ -153,13 +108,14 @@ Content-Type: application/json
...
-| payment             | string | The relative URI of the payment this capture transaction belongs to.                   |
-| capture.id          | string | The relative URI of the created capture transaction.                                   |
-| capture.transaction | object | The object representation of the generic [transaction resource][transaction-resource]. |
+| Property              | Type     | Description                                                                            |
+| :-------------------- | :------- | :------------------------------------------------------------------------------------- |
+| `payment`             | `string` | The relative URI of the payment this capture transaction belongs to.                   |
+| `capture`             | `string` | The capture transaction object.                                                        |
+| └➔&nbsp;`id`          | `string` | The relative URI of the created capture transaction.                                   |
+| └➔&nbsp;`transaction` | `object` | The object representation of the generic [transaction resource][transaction-resource]. |
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]: Include `transaction-response.md` here as well?

File: payments/mobile-pay/after-payment.md [hunk 5]
```diff
@@ -386,83 +206,39 @@ Content-Type: application/json
...
-| payment                  | string | The relative URI of the payment this cancellation transaction belongs to.              |
-| cancellation.id          | string | The relative URI of the current cancellation transaction resource.                     |
-| cancellation.transaction | object | The object representation of the generic [transaction resource][transaction-resource]. |
+| Property              | Type     | Description                                                                            |
+| :-------------------- | :------- | :------------------------------------------------------------------------------------- |
+| `payment`             | `string` | The relative URI of the payment this cancellation transaction belongs to.              |
+| `cancellation`        | `string` | The current cancellation transaction object.                                           |
+| └➔&nbsp;`id`          | `string` | The relative URI of the current cancellation transaction resource.                     |
+| └➔&nbsp;`transaction` | `object` | The object representation of the generic [transaction resource][transaction-resource]. |
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]: Use the include here?

File: payments/mobile-pay/after-payment.md [hunk 6]
```diff
@@ -529,25 +306,28 @@ Content-Type: application/json
...
-| :------------------- | :----- | :------------------------------------------------------------------- |
-| payment              | string | The relative URI of the payment this capture transaction belongs to. |
-| reversal.id          | string | The relative URI of the created capture transaction.                 |
-| reversal.transaction | object | The object representation of the generic transaction resource.       |
+| Property              | Type     | Description                                                                            |
+| :-------------------- | :------- | :------------------------------------------------------------------------------------- |
+| `payment`             | `string` | The relative URI of the payment this capture transaction belongs to.                   |
+| `reversal`            | `string` | The reversal transaction.                                                              |
+| └➔&nbsp;`id`          | `string` | The relative URI of the created capture transaction.                                   |
+| └➔&nbsp;`transaction` | `object` | The object representation of the generic [transaction resource][transaction-resource]. |
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]: Include?

File: payments/mobile-pay/other-features.md [hunk 7]
```diff
@@ -16,6 +16,211 @@ sidebar:
...
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "payment": {
```
Comment (asbjornu) [inline payments/mobile-pay/other-features.md]: Re-indent this code example to 4 spaces, please.

Comment (NullableInt) [reply payments/mobile-pay/other-features.md]: It has 4 spaces here and is committed and pushed.

File: payments/mobile-pay/after-payment.md [hunk 8]
```diff
@@ -332,8 +112,8 @@ You can only cancel a payment - or part of payment - not yet captured.
 **Request**
 
 ```http
-POST /psp/mobilepay/payments/e7919b4f-81a2-4ffb-ec40-08d617d580a2/cancellations HTTP/1.1
-Host: {{ page.apiUrl }}
+POST /psp/mobilepay/payments/{{ page.paymentId }}/cancellations HTTP/1.1
+Host: api.externalintegration.payex.com
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]:
```suggestion
Host: {{ page.apiUrl }}
```

File: payments/mobile-pay/after-payment.md [hunk 9]
```diff
@@ -471,8 +174,8 @@ captured payment.
 **Request**
 
 ```http
-POST /psp/mobilepay/payments/e7919b4f-81a2-4ffb-ec40-08d617d580a2/reversals HTTP/1.1
-Host: {{ page.apiUrl }}
+POST /psp/mobilepay/payments/{{ page.paymentId }}/reversals HTTP/1.1
+Host: api.externalintegration.payex.com
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]:
```suggestion
Host: {{ page.apiUrl }}
```

File: payments/mobile-pay/other-features.md [hunk 10]
```diff
@@ -16,6 +16,211 @@ sidebar:
...
+Property of the JSON document is described in the following section. Use the
+[expand][technical-reference-expand] request parameter to get a response that
+includes one or more expanded sub-resources inlined.
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/mobilepay/payments HTTP/1.1
+Host: api.externalintegration.payex.com
```
Comment (asbjornu) [inline payments/mobile-pay/other-features.md]:
```suggestion
Host: {{ page.apiUrl }}
```

File: payments/mobile-pay/after-payment.md (hunk 8 above)
Comment (asbjornu) [reply payments/mobile-pay/after-payment.md]: ☝️

File: payments/mobile-pay/after-payment.md (hunk 9 above)
Comment (asbjornu) [reply payments/mobile-pay/after-payment.md]: ☝️

File: _includes/transaction-response.md [hunk 11]
```diff
@@ -7,7 +7,7 @@
 
 ```http
 GET /psp/{{ instrument }}/payments/{{ page.paymentId }}/{{ plural }} HTTP/1.1
-Host: api.payex.com
+Host: {{ page.apiUrl }}
```
Comment (asbjornu) [inline _includes/transaction-response.md]:
```suggestion
Host: {{ page.apiHost }}
```

File: payments/mobile-pay/after-payment.md [hunk 12]
```diff
@@ -108,58 +62,21 @@ Content-Type: application/json
...
-```
+the `GET` request.
 
-{:.table .table-striped}
-| Property            | Type   | Description                                                                            |
-| :------------------ | :----- | :------------------------------------------------------------------------------------- |
-| payment             | string | The relative URI of the payment this capture transaction belongs to.                   |
-| capture.id          | string | The relative URI of the created capture transaction.                                   |
-| capture.transaction | object | The object representation of the generic [transaction resource][transaction-resource]. |
+{% include transaction-response.md payment-instrument="mobile-pay"
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]:
```suggestion
{% include transaction-response.md payment-instrument="mobilepay" %}
```

File: payments/mobile-pay/after-payment.md [hunk 13]
```diff
@@ -32,66 +32,20 @@ sidebar:
...
-```http
-GET /psp/mobilepay/payments/e7919b4f-81a2-4ffb-ec40-08d617d580a2/captures HTTP/1.1
-Host: {{ page.apiUrl }}
-Authorization: Bearer <AccessToken>
-Content-Type: application/json
-```
-
-{:.code-header}
-**Response**
+{% include transaction-response.md payment-instrument="mobile-pay" transaction="capture" %}
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]: Shouldn't it be `mobilepay` and not `mobile-pay` as the argument is used in API URLs where `mobilepay` is the correct spelling? Also, when the `transaction` is `capture`, the argument can be removed, no?

```suggestion
{% include transaction-response.md payment-instrument="mobilepay" %}
```

File: payments/mobile-pay/after-payment.md [hunk 14]
```diff
@@ -108,58 +62,21 @@ Content-Type: application/json
...
+the `GET` request.
 
-{:.table .table-striped}
-| Property            | Type   | Description                                                                            |
-| :------------------ | :----- | :------------------------------------------------------------------------------------- |
-| payment             | string | The relative URI of the payment this capture transaction belongs to.                   |
-| capture.id          | string | The relative URI of the created capture transaction.                                   |
-| capture.transaction | object | The object representation of the generic [transaction resource][transaction-resource]. |
+{% include transaction-response.md payment-instrument="mobile-pay"
+    transaction="capture"%}
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]:
```suggestion
```

File: payments/mobile-pay/after-payment.md [hunk 15]
```diff
@@ -169,161 +86,24 @@ authorization amount.
...
-`Authorization` transaction has been created.
-The common use-case for the finalize operation is to authorize the payment
-(that has the preauthorization intent) and complete all payment related
-activities as soon as possible - in order to complete (finalize) everything
-server-to-server afterwards.
-The only allowed activity is `Finalize`. To use the operation, you should
-perform a `GET` on the payment after the user returns from the
-`redirect-authorization` operation and find the operation
-`update-authorization-finalize`.
+{% include transaction-response.md payment-instrument="mobile-pay" transaction="cancellation" %}
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]:
```suggestion
{% include transaction-response.md payment-instrument="mobilepay" transaction="cancellation" %}
```

File: payments/mobile-pay/after-payment.md [hunk 16]
```diff
@@ -346,123 +126,46 @@ Content-Type: application/json
...
+| ✔︎   | └➔&nbsp;`payeeReference` | `string(50)` | A unique reference for the cancellation transaction. See [payeeReference][payee-reference] for details. |
 
 The `cancel` resource contains information about a cancellation transaction
 made against a payment.
 You can return a specific cancellation transaction by adding the transaction
 id to the `GET` request.
 
-{:.code-header}
-**Response**
+{% include transaction-response.md payment-instrument="mobile-pay"
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]:
```suggestion
{% include transaction-response.md payment-instrument="mobilepay"
```

File: payments/mobile-pay/after-payment.md [hunk 17]
```diff
@@ -346,123 +126,46 @@ Content-Type: application/json
...
-```
-
-{:.table .table-striped}
-| Property       | Type   | Description                                                                                          |
-| :------------- | :----- | :--------------------------------------------------------------------------------------------------- |
-| payment        | string | The relative URI of the payment that the reversal transactions belong to.                            |
-| id             | string | The relative URI of the created reversal transaction.                                                |
-| reversalList   | array  | The array of reversal transaction objects.                                                           |
-| reversalList[] | object | The reversal transaction object representation of the reversal transaction resource described below. |
+{% include transaction-response.md payment-instrument="mobile-pay" transaction="reversal" %}
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]:
```suggestion
{% include transaction-response.md payment-instrument="mobilepay" transaction="reversal" %}
```

File: payments/mobile-pay/after-payment.md [hunk 18]
```diff
@@ -487,67 +190,37 @@ Content-Type: application/json
...
+| ✔︎   | └➔&nbsp;`payeeReference` | `string(50)` | A unique reference for the reversal transaction. See [payeeReference][payee-reference] for details.                       |
 
 The `reversal` resource contains information about a reversal transaction made
 against a payment.
 You can return a specific reversal transaction by adding the transaction id to
 the `GET` request.
 
-{:.code-header}
-**Response**
+{% include transaction-response.md payment-instrument="mobile-pay"
```
Comment (asbjornu) [inline payments/mobile-pay/after-payment.md]:
```suggestion
{% include transaction-response.md payment-instrument="mobilepay"
```

**Pull request merged**